README.md

```diff
@@ -184,8 +185,11 @@ Note: Don't use `@` symbol when adding username.
 When you are done with the setup, you should host your website online.
 We highly recommend to read through the [Deploying on Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) docs for React.
 
-#### Configuring GitHub Actions
-- Using the Personal Access Token you placed in the `.env` file earlier create a [repository secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `OPEN_SOURCE_TOKEN` where the value matches the token value from the `.env` file in your local workspace.
+#### Configuring GitHub Actions (Recommended)
+First you should enable, Github Actions for the repository you use.
+
+Their is a [CRON setup](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events) up so that it deploys your site every week, so that once you update your profile on Github it is shown on your portfolio. You can also trigger it manually using `workflow_dispatch` event, see https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch on how to do that.
```

Comment:
Change this to start with: 'A configurable [CRON Job]() is setup which deploys your site....'
Maybe move the linked blog as a link like '...see [this guide]() on how to do that'